typescript/libs/node_client.py

```diff
@@ -111,7 +111,7 @@ def makeTimeoutMsg(self, cmd, seq):
         }
         return timeoutMsg
 
-    def sendCmd(self, cb, cmd, seq):
+    def sendCmd(self, cmd, seq, cb):
```

Comment:
Changed that to match the same order. The original code seemed wrong too, as `seq` appears in the middle. I think it won't affect the functionality as the number of parameters is different, but I agree for consistence they should be kept the same order.